package/MDAnalysis/topology/MOL2Parser.py

```diff
@@ -88,15 +88,44 @@ class MOL2Parser(TopologyReaderBase):
     unknown atom types, they will be assigned an empty element record. If all
     atoms have unknown atom types, the elements attribute will not be set.
 
+    Dealing with optional fields:
+    (1) ``resid`` will set to 1 when not provided.
+    (2) If no atoms have ``resname`` field, resnames attribute will not be set;
```

Comment:
If you want this to be formatted as an enumerated list then you need to use the proper reST formatting.

Reply:
@orbeckst Thank you for the comment. I intended to make an enumerate list, but not very familiar with ReST. When I type the following (according to ReST doc), it didn't output the desired indentation before the 2nd and 3nd (see image).
```
"""
    Dealing with optional fields:
    
    1. ``resid`` will set to 1 when not provided.
        
    2. If no atoms have ``resname`` field, resnames attribute will not be set;
    If some atoms have ``resname`` while some do not,
    :exc:`ValueError` will occur.
    
    3. If "NO_CHARGES" shows up in "@<TRIPOS>MOLECULE" section
    and no atoms have the ``charge`` field, charges attribute will not be set;
    If "NO_CHARGES" shows up while ``charge`` field appears,
    :exc:`ValueError` will occur;
    If a charge model is specified while some atoms don't have ``charge``,
    :exc:`ValueError` will occur as well.
"""
```
![image](https://user-images.githubusercontent.com/19585264/162272330-f28049e1-c36b-46ce-9cf0-86b7db3f572a.png)


However, if without space lines between different points, `build_sphinx` will raise an error. Sorry that I don't know how to make the enumerate list work.
```Warning, treated as error:
MOL2Parser.py:docstring of MDAnalysis.topology.MOL2Parser.MOL2Parser:25:Enumerated list ends without a blank line; unexpected unindent.
```